src/classes/state/state.ts

```diff
@@ -0,0 +1,71 @@
+import {IState} from '../../interfaces/state/state';
+import * as core from '@actions/core';
+import {IIssuesProcessorOptions} from '../../interfaces/issues-processor-options';
+import {IStateStorage} from '../../interfaces/state/state-storage';
+import {IIssue} from '../../interfaces/issue';
+
+export type IssueID = number;
```

Comment:
I want to hide the exact type of because of

1. at the moment i am  not 100% sure `issue.number` is the robust unambiguous issue ID, so there's chance the other issue property will be selected as ID and it can happen to be of some other type. Aliasing the type makes the change painless. This is the 1st benifit. 

2. named type improves readability Set<IssueID> - clearly says about "a set of issue identificator" while Set<number> is nothing but just "a set of numbers of unclear purpose". It works especially good with functions parameters.